### ISSUE TITLE
Errata update: Ch13, page 403, formula for $\beta_{PC, i}$

### ISSUE THREAD

**Issue by esvhd** [rmcelreath/rethinking#73]
This is the **last line** on the page, formula for $\beta_{PC, i}$ in the mathematical model.

Instead of:

$\beta_{PC, i} = \beta_{P}.....$

Should it be:

$\beta_{PC, i} = \beta_{PC}.....$

$\beta_{P}$ is already used by the formula above it for $\beta_{P, i}$.

Hope I'm not missing anything here.

**Comment by wpetry** [comment]
Yep, I agree this is a typo, and I came to the same conclusion for the correction.
